### DISCUSSION TITLE
Maturity and Fecundity

### DISCUSSION THREAD

**Discussion by sabrinabeyer** [pfmc-assessments/shortspine_thornyhead_2023#13] [General]
Starting a discussion thread for maturity and fecundity. @JaneSullivan-NOAA @pyhernvann 

We are expecting updated maturity information for SST from the NWFSC combo survey from Melissa Head by mid-March.

Thanks @JaneSullivan-NOAA for adding the Pearson and Gunderson 2003 maturity curve!

I added code to pull the existing maturity data from the NWFSC combo survey that I think was used in a sensitivity analysis in the 2013 assessment. I didn't know that this was available to us before Kiva mentioned in today. ....I think all of this data will change, so I don't think it's worth our time to go back and redo any of the sensitivities in the 2013 assessment. But it is worth thinking about how to analyze the new data in comparison to the Pearson and Gunderson 2003.

**Comment by sabrinabeyer** [comment]
Quick note-- The SST histology analysis for maturity information is delayed but Melissa Head (NWFSC) will send what she has for SST maturity data by the end of the week for us to start on.

**Comment by sabrinabeyer** [comment]
@JaneSullivan-NOAA 

Here is an initial attempt at looking at the WCGBTS maturity information. I may have something wrong in my script (something to check!!), but there was no difference in the curves if using biological of functional maturity. I only used data with Certainty=1. This is the "biological maturity" curve. Feel free to use this (or some version of this--I am not good with ggplot!) at the data workshop if you think it is helpful. Or keep what you have with the graph and sensitivity from the last assessment and just mention that we received updated maturity information from the WCGBTS from Melissa Head and will explore the updated data in the assessment.

![SST_maturity_comparison](https://user-images.githubusercontent.com/63366861/226057484-f6d1e4c7-71c4-4a6d-8d74-9db9887653b8.png)

**Comment by sabrinabeyer** [comment]
Questions for Melissa Head about the maturity data. Please feel free to add questions!

1. What are the definitions of "biological" and "functional" maturity?
2. What is "Certainty"? Should we filter data by this?
3. Insights into the shape of the curve--gradual slope due to a considerable number of large females being biologically immature. 
4. Other insights into the reproductive biology and which data to consider in the assessment? Spawning season? Any temporal filters to consider?

**Reply by JaneSullivan-NOAA** [reply]
Awesome, thank you @sabrinabeyer 

Below is an excerpt from Pearson and Gunderson (2003). It might be useful to ask Melissa if she used the same or different characteristics to classify gross and histological maturity status.
![image](https://user-images.githubusercontent.com/68303089/226070959-19b87196-f960-472f-b919-b20170d68fa7.png)
![image](https://user-images.githubusercontent.com/68303089/226071103-fadaa1fe-6bf7-4191-8c88-9f56a1c5aca1.png)
![image](https://user-images.githubusercontent.com/68303089/226071122-9b9dbb78-97cf-4224-8c20-8837f3667710.png)

**Reply by sabrinabeyer** [reply]
Very helpful-Thank you Jane!

**Comment by sabrinabeyer** [comment]
I found the error in my code! There is a slight difference between maturity-at-length when considering biological or functional maturity in the direction that we expected. Note, the curve in the presentation is "biological" maturity. Here's the comparison between biological and functional maturity from the WCGBTS.
![image](https://user-images.githubusercontent.com/63366861/226421022-c3aa8f26-a101-4824-a1ca-944b0d8cdfd5.png)

**Reply by sabrinabeyer** [reply]
...I see that I need to update the L50 line. Will do that next.

**Reply by JaneSullivan-NOAA** [reply]
Hey @sabrinabeyer no rush on this! We have submitted our slides already, and I am presenting the results as preliminary

**Comment by sabrinabeyer** [comment]
Maturity team: @JaneSullivan-NOAA @pyhernvann 

What are the next steps for wrapping up the maturity data? I believe there are some decisions to make that we could do as a group or bring up at the SST team meeting to finalize.

The 2013 assessment used the Pearson and Gunderson 2003 maturity-at-length information and did a sensitivity to early information from the WCGBTS. 

We now have a lot more maturity-at-length information from the WCGBTS (provided by M. Head). We still have the same finding of different maturity curves and not a great reason for why. (Possibly because P&G sampled mostly northern CA to WA, and those fish seem to mature at a smaller size? Possibly due to different definitions of maturity--but after re-reading P&G, I think both are using a similar classification. Lots of other unknowns. P&G don't mention mass atresia events-- maybe timing of collections also matters? P&G also did not observe multiple broods. We won't know)

Thoughts?

I know this is not a research project...but some quick plots of the new maturity data don't show much evidence for differences in maturity-at-length by depth, but do show potential differences between north (OR/WA) and south (CA) although range of lengths and sample sizes get pretty slim picking...
![image](https://user-images.githubusercontent.com/63366861/228095544-67b41afc-375f-42f6-9816-eb24eee6f00b.png)
![image](https://user-images.githubusercontent.com/63366861/228095626-79c1d3c9-e440-42b0-b960-8e74fd3cd88a.png)

Question:
Which maturity information to use? Do the same as the 2013 assessment, or use the updated maturity information in the model and the Pearson and Gunderson as a sensitivity? I am a bit confused about what is the most appropriate approach.

**Reply by sabrinabeyer** [reply]
Here was the comparison graph:
![image](https://user-images.githubusercontent.com/63366861/228096407-e8e116c9-c147-4f49-848a-ff4cffa07a94.png)

**Comment by pyhernvann** [comment]
Hi Sabrina,



Sorry I totally dropped the ball on this over the last month – pretty busy
with discard data and personal work

As I missed a lot of things, I allow myself to formulate plenty silly
comments very redundant with the thoughts you already had

-        Thanks for drawing these maturity curves. Based on what we can see
here and in Fig.10 of the 2013 report, it’s tending to include both
latitude and depth in the model that would estimate. All the more so as
this pattern is observed for other species (e.g. darkblotched
<https://doi.org/10.1007/s10641-015-0441-1>) – I don’t know how we could do
it while keeping parsimonious though.

-        I still struggle to picture the data used in Pearson and
Gunderson. It is a shame we don’t have better than the parameter estimates.
So difficult to assess whether the season effect may trigger most of the
differences observed between the two datasets…

-        Once my comment #1 made, what should we do if we fit a model that
includes covariates..? Which combinations of lat/depth should be considered
to produce the maturity curve that will be used in our model..? Those of
the center of gravity of the pop?

-        Well, I don’t know what to do either at the end of the day – the
minimum will be to conduct a sensitivity analysis, of course.

-        If some sampling programs and new studies are conducted on these
aspects in the coming years, for sure the WCGBTS will definitely be more
central in the stock assessment ..and P&G could be dropped? But right now,
I don’t have a super strong opinion


PY


Le lun. 27 mars 2023 à 17:31, sabrinabeyer ***@***.***> a
écrit :

**Reply by sabrinabeyer** [reply]
Thanks @pyhernvann!

I also found this guidance in the "unofficial assessment handbook":

"Data from the NWFSC survey on maturity includes a column indicating mature or immature and another indicating spawning and not spawning. The latter considers all “mature” fish with over 25% atresia as not spawning (along with all immature fish). The spawning/not spawning column is the one we commonly use to estimate the maturity curve since that is really what we care about. In some cases a simple logistic will fit, but if there is much skip spawning/atresia for older/larger females, a logistic type curve which asymptotes to a lower value or a non-parametric fit is more appropriate. A column with percent atresia is also provided if you with to use a percentage other than 25% for the cutoff. Finally, the mature/immature column can be used instead if the atresia/skip spawning is taken into account in specifying the fecundity relationship."

**Comment by pyhernvann** [comment]
Hi mature people,

I have tried to apply a statistical approach for estimating the Length-Mat
rel. parameters while including environmental effects in a relatively
simple way. I can report on that on Thursday. I fear I won't have the time
to meet before.

Have a nice day,
PY

Le mar. 28 mars 2023 à 14:04, sabrinabeyer ***@***.***> a
écrit :

**Comment by pyhernvann** [comment]
Hi @sabrinabeyer & @JaneSullivan-NOAA - I developed a glm approach yesterday for fitting the maturity ogives while including latitude and depth as covariates. Sabrina just sent me the document that Ben Williams produced and we basically applied a very comparable methodology (I actually missed Ben's mail forwarded 12days ago). Given the time I have spent on this I'd be happy to investigate it deeper and propose you a full framework including both Ben's stuff and mine.

**Reply by JaneSullivan-NOAA** [reply]
@pyhernvann you have my full blessing! Thank you for leading the charge. What are your and @sabrinabeyer's thoughts on using the new data/analysis for the base model (i.e., replacing the Pearson and Gunderson curve) vs. using it as a sensitivity? Right now I am hesitant to use these data for the base stock assessment model.

**Reply by pyhernvann** [reply]
I think that the fact that we observe an influence of both depth and latitude in the L50 highlights that the spatial coverage of the samples will have a substantial impact on the parameters we estimate. Although I trust Pearson and Gunderson's work, the tricky thing is that we have no idea of what the data looked like in terms of length or spatial coverage (we only have bounds).
Additionally, the data provided by M. Head includes now more samples (~twice) than what was available in P&G.
So I personally would be inclined to use the data newly available for the base model, and perhaps keep P&G for a sensitivity analysis (i.e., as a supportive argument for dropping this dataset).
Then choosing between a rough estimate of the curve from M. Head's data or a finer method like a GAM- or GLM-based approach will be based (i) on the outcomes of this exploration and (ii) on the degree of satisfaction of the team. The option not kept for the base model would be passed to the sensitivity analysis.

**Reply by sabrinabeyer** [reply]
Thanks @pyhernvann! @JaneSullivan-NOAA - I think this is a good question to ask the instructors.

**Comment by JaneSullivan-NOAA** [comment]
Notes from meeting with O Hamel:
- potentially weight samples proportional to survey biomass
- similarly, could weight be depth strata (but we don't sample their entire depth range and it seems like the depth differences are huge)
- dealing with a sparsity of small fish: does this create an issue with anchoring the ascending limb of the logistic curve? B Williams doesn't seem to think so but maybe
- seasonal issues are likely the biggest factor. need to double check with M Head about the assumptions of the data (she's pretty confident about it) and what her advice is
- recommends using the new M Head data for the base assessment model and using P&G2003 as a sensitivity (plus a second "in-between" sensitivity)
- she found evidence of some batch spawning

**Comment by sabrinabeyer** [comment]
@JaneSullivan-NOAA @Ovec8hkin Sorry to be slow...this is what E.J. responded. 

Hi E.J.,

I am working on the West Coast Rex sole and Shortspine thornyhead stock assessments as part of a UW class. For both species we are looking to add a length-fecundity relationship in Stock Synthesis. In a first pass, our scaling of the biomass was way off when adding the fecundity information and I found this advice in the "unofficial stock assessment" handbook that I think helps us with the scaling . But I wanted to double check with you to make sure this is still correct and is needed because recruits are modeled as recruits in 1000s?
....

image.png

.....

Also, is it a problem if we're using fecundity-length relationships (fecundity as a power function of length) from published studies that do not report a bias correction? (i.e., reports the median, not mean). I am only now wrapping my head around all of this, but wondering if that is a problem or will lead to a bias? 
......
E.J.'s response:

The equation you sent and its description are correct *if* your original fecundity-length units are length in mm and fecundity in eggs (i.e. it matters what units you start from). Given that, the conversion gets you fecundity in 1000s of eggs per fish, but as the document states, recruitment and numbers of fish in SS3 are in units of 1000s of fish, hence fecundity reported by SS3 will be in millions. What are the units of the fecundity-length relationship that you are converting from? My personal preference, FWIW, is to report eggs in billions (i.e. divide by 1e6), since as you know a single fish can produce millions of eggs in many species. Reporting in billions of eggs just makes the spawning output numbers smaller and easier to report in tables, etc.

The bias correction is a multiplicative term in the back-transformation -- exp((sigma^2) / 2) -- so it will rescale the egg output reported by the model. However, that is mathematically equivalent to changing the units and will not have an effect on the dynamics. If the bias transformation was not used in the literature values, it will only change the (now arbitrary) scale of the reported spawning output.
.....

Also see:
https://github.com/EJDick-NOAA/Rockfish-Fecundity

which goes through the conversion for SS3 for rockfish fecundity-at-length relationships, where length is measured in mm (from the Dick et al. 2017 study)


A couple of things:

1) SST already has a fecundity-at-length relationship using length in cm (so we don't need the (a*10^b) conversion from mm to cm)
2) The fecundity parameter currently saved in the R data has the a parameter (intercept) divided by 1000 because SS3 reports numbers in 1000s of recruits. (this was in the "unofficial handbook")
3) HOWEVER, E.J. is suggesting to instead divide by 1000000 so the reported spawning output will be in units of billions of eggs if the fecundity-length relationship is in millions of eggs per fish.
4) I have NOT made this change and I do not know the influence on spawning biomass---a bit worried here!  I will follow-up with E.J. about how this influences the estimate of spawning biomass and if models are comparable when switching to a specified length-fecundity relationship.

I know this needs to be done asap....would you like me to try a model with different scaling of the a parameter? a/1000 and a/1000000?

**Reply by sabrinabeyer** [reply]
woop, the image didn't come through for the conversion in unofficial handbook:

<img width="271" alt="image" src="https://user-images.githubusercontent.com/63366861/230678446-f179be22-8ad0-45a4-a2ff-25e8d02885f7.png">

**Comment by sabrinabeyer** [comment]
E.J.'s follow-up email about the change in spawning output vs spawning biomass when using a length-fecundity relationship in the model. In short, the scaling of the summary spawning biomass should NOT change (this information comes from the catch time series), but the scale of the spawning output (eggs produced) will change if adjusting the scale of the intercept (a) parameter, where fecundity = a*L^b. With spawning output, we are more concerned with the relative changes in spawning output (i.e., spawning depletion) of the population. My understanding of spawning depletion is it is the ratio of number of eggs produced divided by the expected number of eggs from an unfished population (and is somewhat different than SPR).

.....
From E.J.,

Happy to help! Yes, if length is already in cm, then you are just dividing eggs by 1000 (or 1e6, or whatever you like) to change the units. Regarding spawning biomass, there's an unfortunate lack of standardization in terms that confuses everyone. Spawning 'biomass' is often used interchangeably to refer to the mass of mature females, or, the total number of eggs they produce. I like to say "egg production" or "spawning output" when referring to eggs, and "spawning biomass" when referring to the mass of all mature females. As you know, when b=3, they are roughly proportional. So, using those terms, the value of the intercept parameter ("a") in the fec-len relationship will not affect spawning biomass. However, it *does* change the units of spawning output (egg production). The scale of adult biomass is largely determined by the scale of the catch time series, and the conversion from eggs to recruits via the S-R relationship does not rely on a "true" (absolute) scaling of spawning output.

When comparing models, you can compare the spawning biomass (mass of mature females), but not the spawning output (unless the two models have identical biological parameters). That is often why we set the "summary biomass" age in SS3 to age at 50% maturity. That roughly approximates the mass of mature females (or 2x the mass, if males are included and there is a ~50:50 sex ratio and similar growth), and can be more closely compared to other model structures that track mature biomass, e.g. production models like what we used for your cowcod project.

I'd say set the summary biomass age to A50 (or close), then run your models with different "a" values. The scale of the spawning output will change, but the total biomass, summary biomass, and relative biomass trends should not, all else equal.

I think. ;-)

    E.J.
